### ISSUE TITLE
[Accessibility] Screenreader doesn't announce 'button' role for Copilot Chat send button

### ISSUE THREAD

**Issue by gdomingu** [microsoft/vscode#231059]
## Environment Details:
Application Name: VS Code: Copilot Copilot Chat
Visual studio code: 1.94.2 (Universal)
Copilot chat: v0.21.1
Windows 11 Home 23H2

## Actual Result:​
Screenreader (NVDA, VoiceOver, Orca) announces `Send (enter) section` or `Send (enter) group`

## Expected Result:​
Screenreader should announce role. `Send (enter), button, section`

Note: The other buttons on the toolbar announces the button role correctly. 

![Image](https://github.com/user-attachments/assets/dca1d559-f514-4ffe-b29c-703c5347f23b)

**Comment by gdomingu** [comment]
Also related, the screenreader isn't announcing the send button state when disabled. Not sure if we will get that for free when changing the type to button.
